coeus-impl/src/main/resources/org/kuali/kra/datadictionary/krad/common/LandingPageView.xml

```diff
@@ -37,9 +37,19 @@
         <property name="items">
             <list>
                 <bean parent="Uif-Message"
-                      p:messageText="Lorem ipsum dolor sit amet, conse ctetuer adipiscing elit, sed diam nonu mmy nibh euismodt. Lorem ipsum dolor sit amet.Lorem ipsum dolor sit amet, conse ctetuer adipiscing elit, sed diam nonu mmy nibh euismodt. Lorem ipsum dolor sit amet.Lorem ipsum dolor sit amet."/>
+                      p:messageText="Kuali Coeus 6.0 is the first release of several designed to provide a new and more intuitive User Experience."/>
                 <bean parent="Uif-Message"
-                      p:messageText="Lorem ipsum dolor sit amet, conse ctetuer adipiscing elit, sed diam nonu mmy nibh euismodt. Lorem ipsum dolor sit amet.Lorem ipsum dolor sit amet, conse ctetuer adipiscing elit, sed diam nonu mmy nibh euismodt. Lorem ipsum dolor sit amet.Lorem ipsum dolor sit amet."/>
+                      p:messageText="The process for developing a Proposal and accompanying Budget has been streamlined. The design emphasizes ease of 
+                      	use for casual or infrequent users of the system while retaining the full set of powerful features from previous KC releases."/>
+                <bean parent="Uif-Message" p:messageText="Specific Features include:"/>
+                <bean parent="Uif-Message" p:messageText="* Re-organized and intuitive data entry."/>
+                <bean parent="Uif-Message" p:messageText="* Left hand navigation and 'Save and Continue' options for linear navigation through the system. No more 'where do I go next'?"/>
+                <bean parent="Uif-Message" p:messageText="* Quick access to tools: validation, print, copy, medusa, hierarchy management, and budget version management is always one-click away in a Proposal Development Document."/>
+                <bean parent="Uif-Message" p:messageText="* Search while you type for several frequently entered data fields."/>
+                <bean parent="Uif-Message" p:messageText="* Modal-based user interactions that provide context and continuity while performing lookups and entering large amounts of data."/>
```

Comment:
Kuali Coeus 6.0 is the first of its kind designed to improve user experience. ???